po/fr.po

```diff
@@ -1286,9 +1272,9 @@ msgid "Go to a random unread article"
 msgstr "Lire un article au hasard"
 
 #: src/keymap.cpp:125
-#, fuzzy
 msgid "Open URL of article, or entry in URL view. Mark read."
-msgstr "Ouvrir l'article dans le navigateur et le marquer comme lu"
+msgstr "Ouvrir l'article ou l'élément dans le navigateur et le marquer"
+"comme lu"
```

Comment:
The original sentence ends with a full stop. The rest around this phrase, however, do not. So I'm not sure whether to add a full stop to make it consistent with the original sentence or whether to leave it out to be consistent with the surrounding list items.

Reply:
@der-lyse I'd prefer translations not to paper over the problems in the original, but I guess we'll end up removing the period from the original as well. So I'm okay with not changing this

Reply:
There should be a space at the end of the first line for them to get concatenated properly.